cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Node.kt

```diff
@@ -98,7 +102,7 @@ open class Node : IVisitable<Node>, Persistable, LanguageProvider, ScopeProvider
     var comment: String? = null
 
     /** Location of the finding in source code. */
-    @Convert(LocationConverter::class) var location: PhysicalLocation? = null
+    @Convert(LocationConverter::class) override var location: PhysicalLocation? = null
```

Comment:
I'd move the doc to the interface.

Can we add the `@Convert` annotation to the interface? It doesn't make any difference right now but if we would use the interfaces for more things, it could make a difference

Reply:
Moved the doc. Unfortunately, the convert annotation needs toe specified at the field and therefore at the implemented class.